packages/components/number-input/__tests__/number-input.test.tsx

```diff
@@ -240,6 +240,54 @@ describe("NumberInput", () => {
 
     expect(stepperButton).toBeNull();
   });
+
+  it("should clear value when isClearable and pressing ESC key", async () => {
+    const onClear = jest.fn();
+    const defaultValue = 1234;
+
+    const {container} = render(
+      <NumberInput
+        isClearable
+        defaultValue={defaultValue}
+        formatOptions={{useGrouping: false}}
```

Comment:
why we need to specify `formatOptions={{useGrouping: false}}` for this test case?